src/VehicleSetup/SetupView.qml

```diff
@@ -41,11 +41,7 @@ Rectangle {
     function showSummaryPanel()
     {
         if (_fullParameterVehicleAvailable) {
-            if (QGroundControl.multiVehicleManager.activeVehicle.autopilot.vehicleComponents.length == 0) {
```

Comment:
This was a mistake. I think I thought this was an armed check and took it out, then took the component definition further down out afterward after doing ctl+f and not finding any usages. I will fix soon.